### DISCUSSION TITLE
Surprised by Firefox ESR 128

### DISCUSSION THREAD

**Discussion by rebop** [Aris-t2/CustomCSSforFx#683] [Q&A]
Hi folks. Was not expecting Firefox to suggest to me to update from Firefox ESR 155 to 128. Cannot recall nor find which CustomCSS I am using, but the dates are 9/26/23.

More suprising, I do not see anything that needs changing!! Have not looked in real depth, biut will. Is it possible I need to make no changes for 128?

Thanks!

**Comment by rebop** [comment]
Apologies. Moved to the discussions topic and found one small issue.

**Reply by Aris-t2** [reply]
I will close this then. 
https://github.com/Aris-t2/CustomCSSforFx/discussions/606#discussioncomment-10822239
